server.js

```diff
@@ -0,0 +1,204 @@
+const express = require('express');
+
+const app = express();
+const bodyParser = require('body-parser');
+const path = require('path');
+const Sequelize = require('sequelize');
+const dbConnect = require('./models/db');
+const whenDbCreated = require('./models/whenDbCreated');
+const Task = require('./models/Task')
+const User = require('./models/User')
+const Priority = require('./models/Priority')
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+whenDbCreated.then(() => {
+  User.findAll({
+    raw: true
+  }).then(users => {
+    console.log("USER_ID: ", users);
+  });
+});
+
+
+app.get('/api/store/',(req,res)=>{
+  console.log(req.params.userId)
+  whenDbCreated.then(() => {
```

Comment:
Unnecessary call to `whenDbCreated`. When call to `/api/store/` endpoint is made `User/Task/Priority` models are already created.